system_tests/federation_system_test.go

```diff
@@ -51,7 +64,7 @@ var _ = Describe("federation", func() {
 
 		Expect(upstream.Name).To(Equal(federation.Spec.Name))
 		Expect(upstream.Vhost).To(Equal(federation.Spec.Vhost))
-		Expect(upstream.Definition.Uri).To(Equal(federation.Spec.Uri))
+		Expect(upstream.Definition.Uri).To(Equal(federationUri))
```

Comment:
I'm surprised this passes, given one is a string and the other a `corev1.Secret`. Am I missing something?

Reply:
Ah I see, it's a naming collision with the string in line 21, being redefined in line 25. Could you rename the secret please?

Reply:
Updated the name of the secret.